SpaceWarpMod/UI/ModuleManagerWindow.cs

```diff
@@ -127,6 +129,9 @@ public class ModuleManagerWindow : ResizableWindow {
                     GUILayout.Label($"{process.Name} ({process.State})", GUILayout.ExpandWidth(true));
                     switch (process.State) {
                     case KontrolSystemProcessState.Available:
+                        if (GUILayout.Button("O", GUILayout.Width(30))) {
```

Comment:
Would it be possible to hide this button if there are no arguments?
Also the label could probably be the number of arguments.